.github/workflows/update-translations.yaml

```diff
@@ -0,0 +1,28 @@
+name: "update translations"
+on:
+  push:
```

Comment:
this should be on a schedule once a week. add to the existing package job?

Reply:
move to it's own step in the `repo-maintenance` workflow